File: hash_map.cpp
#include <iostream>
#include <unordered_map>
using namespace std;

int main() {
    unordered_map<string, string> hash_map = {{"name", "Alice"}, {"city", "Paris"}};
    
    cout << hash_map["name"] << endl;
    
    hash_map["name"] = "Bob";
    
    cout << hash_map["name"] << endl;
    cout << hash_map["city"] << endl;
    
    return 0;
}

File: task2/hash_map.java

import java.util.HashMap;

public class HashTableDemo {
    public static void main(String[] args) {
        HashMap<String, String> hash_map = new HashMap<>();
        hash_map.put("name", "Alice");
        hash_map.put("city", "Paris");
        
        System.out.println(hash_map.get("name"));
        
        hash_map.put("name", "Bob");
        
        System.out.println(hash_map.get("name"));
        System.out.println(hash_map.get("city"));
    }
}
